tests/database.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Replace the contents of this with the database.json file that has more accounts to play around with please. Remember to keep pulling from the main branch to see any changes.